fossa/README.md

```diff
@@ -0,0 +1,13 @@
+# FOSSA
```

Comment:
Is this currently in use? And if not, do we intend to use it in the future?

Reply:
Not currently in use

Reply:
@anvega do we intend to use this in the future? If not, can we unlink it? And if we do, can you help to fill in the rest of the information here?